vowpalwabbit/options_boost_po.cc

```diff
@@ -131,6 +131,12 @@ void options_boost_po::add_and_parse(const option_group_definition& group)
   }
 }
 
+bool options_boost_po::add_parse_and_check_necessary(const option_group_definition& group)
```

Comment:
what is the code formatting complain in this file? @jackgerrits, I checked the log and just the header is referenced